lib/bom.js

```diff
@@ -223,6 +223,11 @@ declare var NodeFilter: {
     SHOW_DOCUMENT_FRAGMENT: number;
 };
 
+declare class CustomEvent extends Event {
+  static initCustomEvent(type: DOMString, canBubble: Boolean, cancelable: Boolean, detail: Object): CustomEvent;
+  detail: Object;
```

Comment:
i think this should be `any`. it sounds like it could be non-objects like strings or null too